packages/devtools_app/lib/src/inspector/inspector_data_models.dart

```diff
@@ -370,6 +370,9 @@ class FlexLayoutProperties extends LayoutProperties {
   final VerticalDirection verticalDirection;
   final TextBaseline textBaseline;
 
+  @override
+  List<LayoutProperties> get children => flipMainAxis ? super.children.reversed.toList() : super.children;
```

Comment:
maybe leave children in the real order and add a second getter called 
displayChildren that are in the display order? You should be able to rename children n the parent class to displayChildren and then add a new children getter to the parent class to make this change without having to do much manual work.